CMakeLists.txt

```
@@ -31,6 +31,9 @@ if(SUPPRESS_3RD_PARTY_WARNINGS)
     set(SUPPRESS_WARNINGS SYSTEM)
 endif()
 
+set(CMAKE_CXX_STANDARD 11)
```

```suggestion
set(CMAKE_CXX_STANDARD 11 CACHE)
```

lets let users overwrite this themselves too?